docs/src/main/asciidoc/command-mode-reference.adoc

```diff
@@ -55,22 +56,28 @@ import io.quarkus.runtime.Quarkus;
 import io.quarkus.runtime.annotations.QuarkusMain;
 
 @QuarkusMain
-public class JavaMain {
+public class HelloWorldMain2 implements QuarkusApplication {
 
     public static void main(String... args) {
-        Quarkus.run(HelloWorldMain.class, args);
+        Quarkus.run(HelloWorldMain2.class, args);
+    }
+
+    @Override
+    public int run(String... args) {
+        System.out.println("Hello World");
+        return 0;
     }
```

Comment:
This is the big change: Instead of requiring both HelloWorldMain and JavaMain for a hello world, just having a single class, HelloWorldMain2 is enough.